lib/features/hooks/hooks.rb

```diff
@@ -148,6 +148,11 @@
     msg = "#{Server.invalid_requests.length} invalid request(s) received during scenario"
     scenario.fail msg
   end
+
+  # Request arrays are cleared after the scenario to allow requests to be registered
```

Comment:
Should we just clear everything here so we don't need this comment?

Reply:
I think I was being overly averse to change risk and it would make a lot more sense for it all to be together.  In fact, having looked again I've realised that I put it in the wrong `After` hook, which would cause problems if the scenario was forced to fail.